### PR TITLE
PXB-3410 [DOC] Add information on PRO builds to PXB 8.4 docs

### PR DESCRIPTION
**Pull request by alina-derkach-oaza**
new file:   docs/install-pro.md
	new file:   docs/pro-files.md
	new file:   docs/pxb-pro.md
	modified:   mkdocs-base.yml
	new file:   snippets/pro-build-announcement.md

### CODE REVIEW

File: docs/install-pro.md [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+            $ sudo percona-release enable-only tools
+            ```
+
+        3. Install *Percona XtraBackup* by running:
+
+            ```{.bash data-prompt="$"}
+            $ sudo yum install percona-xtrabackup-pro
+            ```
+
+            !!! warning
```
Comment (patrickbirch) [inline docs/install-pro.md]: We don't need this warning about CentOS 6. The PXB 8.4 version is not released for CentOS 6.

File: docs/install-pro.md [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+
+This document provides guidelines how to install Pro packages of Percona XtraBackup from Percona repositories. [Check files in packages built for Percona XtraBackup Pro :material-arrow-right:](pro-files.md){.md-button}
+
+## Procedure
+
+1. Request the access to the pro repository from Percona Support. You will receive the client ID and the access token which you use when downloading the packages.
+
+3. Configure the repository and install Percona XtraBackup packages
+
+    === "On Debian and Ubuntu"
```
Comment (patrickbirch) [inline docs/install-pro.md]: Should it be
On Debian or Ubuntu
? 
Same for RHEL or derivatives

File: docs/install-pro.md (hunk 1 above)
Comment (alina-derkach-oaza) [reply docs/install-pro.md]: removed

File: docs/install-pro.md (hunk 2 above)
Comment (alina-derkach-oaza) [reply docs/install-pro.md]: fixed

File: docs/install-pro.md [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+            ```{.bash data-prompt="$"}
+            $ wget https://repo.percona.com/apt/percona-release_latest.$(lsb_release -sc)_all.deb
+            ```
+
+        2. Install the downloaded package with **dpkg**. To do that, run the following commands as root or with **sudo**: `dpkg -i percona-release_latest.$(lsb_release -sc)_all.deb`
+   
+            Once you install this package the Percona repositories should be added. You
+            can check the repository setup in the `/etc/apt/sources.list.d/percona-release.list` file.
+
+        3. Enable the repository: `percona-release enable-only tools release`
```
Comment (mohitj1988) [inline docs/install-pro.md]: We do not release packages in "tools" release repository anymore.
The last release that went in this repo was PXB-8.3.0

Comment (mohitj1988) [reply docs/install-pro.md]: Created this ticket to fix already published 8.4 documentation: 
https://perconadev.atlassian.net/browse/PXB-3424

Page to be fixed: https://docs.percona.com/percona-xtrabackup/8.4/apt-repo.html

File: docs/install-pro.md [hunk 4]
```diff
@@ -0,0 +1,97 @@
...
+        4. Refresh the local cache to update the package information:
+
+            ```{.bash data-prompt="$"}
+            $ sudo apt update
+            ```
+
+        5. Install the `percona-xtrabackup-pro` package:
+
+            ```{.bash data-prompt="$"}
+            $ sudo apt install percona-xtrabackup-pro
```
Comment (mohitj1988) [inline docs/install-pro.md]: I see the package name is "percona-xtrabackup-pro-84"

sudo apt-cache search percona-xtrabackup
percona-xtrabackup-pro-84 - Open source backup tool for InnoDB and XtraDB
percona-xtrabackup-pro-dbg-84 - Debug symbols for Percona XtraBackup
percona-xtrabackup-pro-test-84 - Test suite for Percona XtraBackup
percona-xtrabackup-test-pro-84 - Test suite for Percona XtraBackup
percona-xtrabackup-84 - Open source backup tool for InnoDB and XtraDB

sudo apt install percona-xtrabackup-pro
Reading package lists... Done
Building dependency tree... Done
Reading state information... Done
E: Unable to locate package percona-xtrabackup-pro

File: docs/install-pro.md [hunk 5]
```diff
@@ -0,0 +1,97 @@
...
+            ```{.bash data-prompt="$"}
+            $ sudo yum install \
+            https://repo.percona.com/yum/percona-release-latest.\
+            noarch.rpm
+            ```
+
+        2. Enable the repository: 
+
+            ```{.bash data-prompt="$"}
+            $ sudo percona-release enable-only tools release
```
Comment (mohitj1988) [inline docs/install-pro.md]: We need to remove this

File: docs/install-pro.md [hunk 6]
```diff
@@ -0,0 +1,97 @@
...
+            If *Percona XtraBackup* is intended to be used in combination with
+            the upstream MySQL Server, you only need to enable the `tools repository: 
+
+            ```{.bash data-prompt="$"}
+            $ sudo percona-release enable-only tools
+            ```
+
+        3. Install *Percona XtraBackup* by running:
+
+            ```{.bash data-prompt="$"}
```
Comment (mohitj1988) [inline docs/install-pro.md:81]: refer above

File: docs/install-pro.md [hunk 7]
```diff
@@ -0,0 +1,97 @@
...
+            $ sudo percona-release enable-only tools release
+            ```
+
+            If *Percona XtraBackup* is intended to be used in combination with
+            the upstream MySQL Server, you only need to enable the `tools repository: 
+
+            ```{.bash data-prompt="$"}
+            $ sudo percona-release enable-only tools
```
Comment (mohitj1988) [inline docs/install-pro.md]: refer above

File: docs/install-pro.md [hunk 8]
```diff
@@ -0,0 +1,97 @@
...
+            ```
+
+        3. Install *Percona XtraBackup* by running:
+
+            ```{.bash data-prompt="$"}
+            $ sudo yum install percona-xtrabackup-pro
```
Comment (mohitj1988) [inline docs/install-pro.md]: refer above

Comment (alina-derkach-oaza) [reply docs/install-pro.md]: renamed

File: docs/install-pro.md (hunk 4 above)
Comment (alina-derkach-oaza) [reply docs/install-pro.md]: renamed

File: docs/install-pro.md (hunk 3 above)
Comment (alina-derkach-oaza) [reply docs/install-pro.md]: updated

**Review by mohitj1988 — APPROVED** [review]
Looks good

**Review by patrickbirch — APPROVED** [review]

**Review by satya-bodapati — APPROVED** [review]
LGTM!

**Pull request merged**